templates/index.html

```diff
@@ -33,6 +33,35 @@
   </header>
 
   <div id="builds">
+
+    <!-- B2GDroid promo -->
+    <!--  Spotlight Activity Tag: Mozilla (6247) | Mozilla B2G Activity Tag 2 (52162) | Mozilla Spots (4669) | Expected URL:  -->
+    <script>
```

Comment:
We already have a JS file for code; JS should go in https://github.com/mozilla/nocturnal/blob/master/js/nightly.js